ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearOptimisationEngine.java

```diff
@@ -96,6 +99,7 @@ void run(LinearRaoData linearRaoData, LinearRaoParameters linearRaoParameters) {
         solveLinearProblem();
         if (getSolverResultStatusString().equals("OPTIMAL")) {
             fillCracResults(linearRaoProblem, linearRaoData);
+            fillVirtualCostInCracResult(linearRaoProblem, linearRaoData, linearRaoParameters);
```

Comment:
Here I don't think you need to add linearRaoProblem as a parameter of this method because it is already an attribute of the class (same remark for the previous method)

Reply:
I followed the example of fillCracResults. I think they need it because they are "static" method.